### DISCUSSION TITLE
[2기-A] 남광운

### DISCUSSION THREAD

**Discussion by Namkwangwoon** [super-corini/programmers_kdt_II#359] [2기 Monthly Project I]
# 결과물
- https://github.com/Namkwangwoon/EDA_project
- https://soccer-player-1920.herokuapp.com

# 해결 중 어려웠던 점 및 극복 방법
- Flask나 Django를 사용해서 웹을 다루는것 조차도 부담스럽고 어려웠었다.
- 하지만 새로 알게된 Streamlit을 통해 배포까지도 "비교적" 수월하게 할 수 있었다.

# 새로 공부한 것들
- Streamlit: Machine Learning과 Data Science를 위한 오픈소스 앱 프레임워크이다. (간편하다 + 파이썬을 이용한다)

**Comment by super-corini** [comment]
처음이니 당연히 힘드셨을겁니다. 광운님의 남은 개발인생에서도 새로운 기술을 받아들여야하는일이 적어도 2번 이상 있으실거구요.
이때 이번에 했던 삽질을 바탕으로 광운님이 어떤 스타일로 학습하는 사람인지 인지하시고, 이에 대응할 수 있는 학습을 진행하시면 이번 경험보다 더욱 빠르게 습득 가능하실거에요! 👍

**Reply by Namkwangwoon** [reply]
네 감사합니다!! 😭😭
